nix/mobile/android/release.nix

```diff
@@ -81,7 +83,8 @@ in stdenv.mkDerivation rec {
   ANDROID_NDK_ROOT = "${androidPkgs}/ndk-bundle";
 
   # Used by the Android Gradle build script in android/build.gradle
-  STATUS_GO_ANDROID_LIBDIR = "${status-go}";
+  STATUS_GO_ANDROID_ALL_LIBDIR = "${status-go-android-all}";
```

Comment:
So we get rid of `STATUS_GO_ANDROID_LIBDIR` but not remove reference to it from Gradle config?